site/content/en/docs/Tasks/build_metadata.md

```diff
@@ -0,0 +1,336 @@
+---
+title: "Build Metadata"
+linkTitle: "Build Metadata"
+weight: 5
+date: 2024-01-10
+description: >
+  Adding build information to labels or annotations.
+---
+
+Kustomize build information can be added to resource labels or annotations with the [`buildMetadata`] field.
+
+## Add Managed By Label
+
+Specify the `managedByLabel` option in the `buildMetadata` field to mark the resource as having been managed by kustomize.
+
+The following example adds the `app.kubernetes.io/managed-by` label to a resource.
+
+1. Create a Kustomization file.
+```yaml
+# kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- service.yaml
+buildMetadata: [managedByLabel]
+```
+
+2. Create a Service manifest.
+```yaml
+# service.yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: myService
+spec:
+  ports:
+  - port: 7002
+```
+
+3. Add label with `kustomize build`.
+
+The output shows that the `managedByLabel` option adds the `app.kubernetes.io/managed-by` label to provide information about the resource manifest.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: myService
+  labels:
+    app.kubernetes.io/managed-by: kustomize-v5.2.1
+spec:
+  ports:
+  - port: 7002
+```
+
+## Add Origin Annotation with Local Resource
+
+Specify the `originAnnotations` option in the `buildMetadata` field to annotate resources with information about their origin.
+
+The possible fields of these annotations are:
+
+- `path`: The path to a resource file itself.
+- `ref`: If from a remote file or generator, the ref of the repo URL.
+- `repo`: If from a remote file or generator, the repo source.
+- `configuredIn`: If a generated resource, the path to the generator config. If a generator is invoked via a field
+in the kustomization file, this would point to the kustomization file itself.
+- `configuredBy`: If a generated resource, the ObjectReference of the generator config.
```

Comment:
```suggestion
- `configuredIn`: for generated resources, the path to the generator config. If a generator is invoked via a field
in the kustomization file, this would point to the kustomization file itself.
- `configuredBy`: for generated resources, the ObjectReference of the generator config.
```

Same question for "config" here (although I reckon that "config" is a fairly standardized term).

Reply:
Same as above. I removed the abbreviations and reworded the descriptions.